src/app/pages/OnDemandTvPage/OnDemandTvPage.tsx

```diff
@@ -112,7 +112,16 @@ const OnDemandTvPage = ({
     ? `${brandTitle} - ${episodeTitle} - ${brandName}`
     : headline;
 
-  const episodePageIdentifier = `${service}.bbc_${service}_tv.tv.${episodeId}.page`;
+  const mediaOverrides: MediaOverrides = {
+    model: {
+      language,
+      pageIdentifierOverride: `${service}.bbc_${service}_tv.tv.${episodeId}.page`,
+      pageTitleOverride: promoBrandTitle,
```

Comment:
Does this also work in the case of viewing an episode, rather than a brand page? I realise that we've been testing mostly with a TV Brand page, but also need to make sure that episodes continue to work as expected

Reply:
I've tested this via the Uzbek episode below and can confirm that
the configuration is built accordingly and the player works.
http://localhost.bbc.com:7080/uzbek/bbc_uzbek_tv/tv/w3ct6c2g

Reply:
Excellent, thank you for checking!